design.html

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+title: Design
+id: design
+---
+<div class="row mt-4">
+    <div class="col-md-8 offset-md-2">
+        <div class="box">
+            <h3>Garage Design Hints</h3>
+            <p>The design aims to be very easy to follow and not impose obstacles or constraints on the user.</p>
+            <ul>
+                <li>The font is <a href="https://fonts.google.com/specimen/JetBrains+Mono?preview.text=Garage%20trip" target="_blank">JetBrains Mono ExtraBold 800</a>, which is widely available (e.g. Google Fonts, Google Slides)</li>
+                <li>Prefer lowercase</li>
+                <li>Colors:
+                    <ul>
+                        <li>The primary color is <span class="monospace">#d0e0f4</span> (on dark backrounds or <span class="monospace">#????</span> light backgrounds)</li>
+                        <li>The secondary color is <span class="monospace secondary-color">#5865f2</span> (irrespective of the background)</li>
+                    </ul>
+                </li>
+                <li>The logo for Garage as the whole organization is <span class="monospace">garage_<span class="secondary-color">&#10033;</span>;</span> or the short version <span class="monospace">g_<span class="secondary-color">&#10033;</span>;</span></li>
+                <li>The asterisk may stand for anything and the concept supports nesting like this: <span class="monospace">garage_<span class="secondary-color">trip</span>_<span class="secondary-color">x.y</span>_<span class="secondary-color">night_game</span>;</span></li>
+                <li>Be creative and playful but do not overcomplicate</li>
```

Comment:
Not sure if its required or no, but at first glance I did not even noticed that there is semicolon. So maybe playfully remind it?
```suggestion
                <li>Avoid syntax error, end expression with <span class="monospace">;</span></li>
                <li>Be creative and playful but do not overcomplicate</li>
```